sw/ground_segment/tmtc/server.ml

```diff
@@ -749,9 +749,10 @@ let listen_intruders = fun log ->
   ignore(Ground_Pprz.message_bind "INTRUDER" (update_intruder log))
 
 let send_config = fun http _asker args ->
-  let ac_id' = PprzLink.string_assoc "ac_id" args in
+  let real_id' = PprzLink.string_assoc "ac_id" args in
+  let ac_id' = real_id' in
```

Comment:
Use `real_id` and `ac_id`, no need for the `'` anymore.
Just replace the `ac_id'` at the end by `real_id`.

Reply:
Ok, I did it.